### PR TITLE
QA-60 | Android UI tests on CI

### PR DESCRIPTION
**Pull request by eugenezhernakov**
* Modify & update existing config for running the E2E tests on Firebase via CircleCI
* Configure scheduled workflow to run periodically 
* Setup Slack notifications
* Setup Sauce Labs config just in case we need to test on any devices not available on Firebase

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -140,6 +223,12 @@ jobs:
           command: |
             gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
             gcloud --quiet config set project ${GOOGLE_PROJECT_ID}
+      - run:
+          name: Export environment variables
+          command: |
+            export JUDO_ID=${JUDO_ID} \
+            export API_TEST_TOKEN=${API_TEST_TOKEN} \
+            export API_TEST_SECRET=${API_TEST_SECRET} \
```
Comment (dave-pollock) [inline .circleci/config.yml:231]: I don't understand the purpose of this as it just sets the environment variables to the same value they already have?

Comment (dave-pollock) [reply .circleci/config.yml:231]: The project configuration here sets the environment variables for the pipeline: https://app.circleci.com/settings/project/github/Judopay/JudoKit-Android/environment-variables

It shouldn't be necessary to set them again. I think if you remove it everything should still work fine.

Comment (eugenezhernakov) [reply .circleci/config.yml:231]: No worries - wasn't sure at the time! Cheers, I'll remove from the config now.

File: .circleci/config.yml [hunk 2]
```diff
@@ -235,7 +331,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - feature/QA-60-ui-tests-on-ci
```
Comment (dave-pollock) [inline .circleci/config.yml:328]: Do you want to change this back to master before merging?

Comment (eugenezhernakov) [reply .circleci/config.yml:328]: Yes 😄 This was the last thing I had planned to update after everything else was reviewed.

File: .circleci/config.yml [hunk 3]
```diff
@@ -273,3 +363,16 @@ workflows:
...
+      - build_sample:
+          context: shared-secrets
+      - instrumented_test_sample:        
+          context: shared-secrets
+          requires:
+            - build_sample
+          filters:
+            branches:
+              only:
+                - feature/QA-60-ui-tests-on-ci
```
Comment (dave-pollock) [inline .circleci/config.yml:378]: This too

**Review by dave-pollock — APPROVED** [review]

**Pull request merged**